src/urlConfig.js

```diff
@@ -0,0 +1 @@
+export const baseUrl = "http://localhost:3000";
```

Comment:
```suggestion
export const BASE_URL = "http://localhost:3000";
```
Những thứ gì bất di bất dịch thì đặt tên theo kiểu này nhé